app/main/controllers/template/rtmedia-actions.php

```diff
@@ -1110,5 +1110,24 @@ function rtmedia_hidden_field() {
 		<input type="hidden" name="media_type" value="<?php echo esc_attr( $media_type ); ?>" />
 	<?php
 	}
+
+	/**
+	 * Adds hidden field when rtMedia Membership plugin is active.
+	 * This hidden field sends the album ID necessary for media search
+	 * to work properly inside albums.
+	 *
+	 * @since 28/06/2017
+	 * @link http://git.rtcamp.com/rtmedia/rtMedia/issues/186
+	 * Check issue number 14 in the above link.
+	 */
+
+	global $rtmedia_query;
+
+	// is_plugin_active() won't work without including this file.
+	include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+
+	if ( $rtmedia_query->is_album() && is_plugin_active( 'rtmedia-membership/index.php' ) ) : ?>
+		<input class="rtmedia-current-album" type="hidden" name="rtmedia-current-album" value="<?php echo esc_attr( $media_type ) ?>" />
```

Comment:
please check note in instruction its recommended, we should not trust implications :D 
```
The closing tag of a PHP block at the end of a file is optional, and in some cases omitting it is helpful when using include or require, so unwanted whitespace will not occur at the end of files, and you will still be able to add headers to the response later. It is also handy if you use output buffering, and would not like to see added unwanted whitespace at the end of the parts generated by the included files.
```